src/logger/POSTLoggerOptions.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from './LogLevel';
+
+export default interface POSTLoggerOptions {
+  /**
+   * URL to send logs.
+   */
+  url: string;
+
+  /**
+   * Log batch size. Default value is 85.
+   */
+  batchSize?: number;
+
+  /**
+   * You can use `headers` to provide credentials when `POSTLogger` makes an HTTP POST request.
+   * `POSTLogger` adds `headers` to all requests.
+   */
+  headers?: Record<string, string>;
+
+  /**
+   * Log sending interval. Default value is 2000ms.
+   */
+  intervalMs?: number;
+
+  /**
+   * Level of logging. Check [[LogLevel]] for more information.
```

Comment:
Specify the default value. `LogLevel.WARN` as in other comments

Reply:
Updated it thanks for the catch!